File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/sqlserver/BinaryTypes.kt
package dev.typr.foundations.docs.sqlserver

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class BinaryTypes {
    //start
    val binaryType: SqlServerType<ByteArray> = SqlServerTypes.binary(16)
    val varbinaryType: SqlServerType<ByteArray> = SqlServerTypes.varbinary(255)
    val varbinaryMax: SqlServerType<ByteArray> = SqlServerTypes.varbinaryMax
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/BinaryTypes.scala
package dev.typr.foundations.docs.sqlserver

import dev.typr.foundations.{SqlServerType, SqlServerTypes}

@SuppressWarnings(Array("unused"))
object BinaryTypes:
  //start
  val binaryType: SqlServerType[Array[Byte]] = SqlServerTypes.binary(16)
  val varbinaryType: SqlServerType[Array[Byte]] = SqlServerTypes.varbinary(255)
  val varbinaryMax: SqlServerType[Array[Byte]] = SqlServerTypes.varbinaryMax
  //stop
